Rakefile

```diff
@@ -29,7 +23,7 @@ end
 
 desc 'Make all plugins executable'
 task :make_bin_executable do
-  `chmod -R +x bin/*.rb`
+  `chmod -R +x bin/*`
```

Comment:
Sensu checks are supposed to use file extensions, for Windows compatability. (Yes, I know no one will be able to run the LVM plugin on Windows but :shrug:).